main.go

```diff
@@ -90,6 +94,7 @@ type Config struct {
 	ConnectTo              url.URL           `default:"unix:///var/lib/networkservicemesh/nsm.io.sock" desc:"url to connect to" split_words:"true"`
 	MaxTokenLifetime       time.Duration     `default:"10m" desc:"maximum lifetime of tokens" split_words:"true"`
 	RegistryClientPolicies []string          `default:"etc/nsm/opa/common/.*.rego,etc/nsm/opa/registry/.*.rego,etc/nsm/opa/client/.*.rego" desc:"paths to files and directories that contain registry client policies" split_words:"true"`
+	IPAMPolicy             ipamPolicy        `default:"default" desc:"defines NSE's IPAM Policy. Possible values: default, strict" split_words:"true"`
```

Comment:
Hmm, what if we will use this?
```suggestion
	IPAMPolicy             ipamPolicy        `default:"polite" desc:"defines NSE's IPAM Policy. Possible values: polite, strict" split_words:"true"`
```

Also, I think we should add some notes about policies.